File: app/src/main/java/com/example/easycontacts/example/Contacts.java
package com.example.easycontacts.example;

import com.example.easycontacts.model.Address;
import com.example.easycontacts.model.Contact;
import com.example.easycontacts.model.Email;
import com.example.easycontacts.model.Phone;

import java.util.ArrayList;
import java.util.Date;
import java.util.List;

/**
 * Created by xkrej63 on 29.05.2018.
 */

public class Contacts {

    public static final List<Contact> contacts = new ArrayList<Contact>() {{
        add(new Contact(
                "exapmle-uuid-1",
                "Jan",
                "Novak",
                new ArrayList<Email>() {{
                    add(new Email("personal", "jan.novak@seznam.cz"));
                    add(new Email("work", "jan.novak@work.cz"));
                }},
                new ArrayList<Address>() {{
                    add(new Address("home", "nám. Winstona Churchilla 1938/4, 130 67 Praha 3"));
                }},
                new ArrayList<Phone>() {{
                    add(new Phone("personal", "+420603414256"));
                }},
                "best friend",
                "Best company inc."
        ));
        add(new Contact(
                "exapmle-uuid-1",
                "Helena",
                "Novakova",
                new ArrayList<Email>() {{
                    add(new Email("personal", "helca.novakova@gmail.com"));
                }},
                null,
                new ArrayList<Phone>() {{
                    add(new Phone("personal", "+420608254365"));
                    add(new Phone("work", "+420777254125"));
                }},
                "best friend\nforever",
                "Best company inc."
        ));
    }};
}

File: app/src/main/java/com/example/easycontacts/example/Contacts.kt
package com.example.easycontacts.example

import com.example.easycontacts.model.api.Address
import com.example.easycontacts.model.api.Contact
import com.example.easycontacts.model.api.Email
import com.example.easycontacts.model.api.Phone

import java.util.ArrayList

/**
 * Created by xkrej63 on 29.05.2018.
 */

object Contacts {

    val contacts: List<Contact> = object : ArrayList<Contact>() {
        init {
            add(Contact(
                    "exapmle-uuid-1",
                    "Jan",
                    "Novak",
                    object : ArrayList<Email>() {
                        init {
                            add(Email("personal", "jan.novak@seznam.cz"))
                            add(Email("work", "jan.novak@work.cz"))
                        }
                    },
                    object : ArrayList<Address>() {
                        init {
                            add(Address("home", "nám. Winstona Churchilla 1938/4, 130 67 Praha 3"))
                        }
                    },
                    object : ArrayList<Phone>() {
                        init {
                            add(Phone("personal", "+420603414256"))
                        }
                    },
                    "best friend",
                    "Best company inc."
            ))
            add(Contact(
                    "exapmle-uuid-1",
                    "Helena",
                    "Novakova",
                    object : ArrayList<Email>() {
                        init {
                            add(Email("personal", "helca.novakova@gmail.com"))
                        }
                    },
                    listOf(),
                    object : ArrayList<Phone>() {
                        init {
                            add(Phone("personal", "+420608254365"))
                            add(Phone("work", "+420777254125"))
                        }
                    },
                    "best friend\nforever",
                    "Best company inc."
            ))
        }
    }
}
